base/standard/src/main/java/org/openscience/cdk/tools/manipulator/AtomContainerManipulator.java

```diff
@@ -277,26 +304,33 @@ private static int getAtomicNum(IElement atom) {
 
     /**
      * Calculate the mass of a molecule, this function takes an optional
-     * 'mass flavour' that switches the computation type, the flavours are:
+     * 'mass flavour' that switches the computation type. The key distinction
+     * is how specified/unspecified isotopes are handled. A specified isotope
+     * is an atom that has either {@link IAtom#setMassNumber(Integer)}
+     * or {@link IAtom#setExactMass(Double)} set to non-null and non-zero.
+     * <br>
+     * The flavours are:
      * <br>
      * <ul>
-     *     <li>{@link #MolWeight} (default) - use and isotopes the natural mass
-     *     unless a specific isotope is specified</li>
-     *     <li>{@link #AverageWeight} - use and isotopes the natural mass even
-     *     if a specific isotope is specified</li>
-     *     <li>{@link #MonoIsotopic} - use and isotopes the major isotope mass
-     *     even if a specific isotope is specified</li>
-     *     <li>{@link #MostAbundant} - use the distribution of isotopes
-     *     based on their abundance and select the most abundant. For example
-     *     C<sub>6</sub>Br<sub>6</sub> would have three <sup>79</sup>Br and
-     *     <sup>81</sup>Br because their abundance is 51 and 49%.
+     *     <li>{@link #MolWeight} (default) - uses the exact mass of each
+     *     atom when specified, if not specified the average mass of the element
+     *     is used.</li>
+     *     <li>{@link #MolWeightIgnoreSpecified} - uses the average mass of each
+     *     element ignoring any exact mass that has been specified.</li>
```

Comment:
element ignoring any exact mass that has been specified
=> element, ignoring any isotopic/exact mass specification
(optional suggestion - currently no mention of isotopes)